### ISSUE TITLE
some pages not properly rendered locally

### ISSUE THREAD

**Issue by rmsrosa** [rmsrosa/booksjl-franklin-template#8]
For some reason, Franklin is not rendering properly some pages when serving/building them locally.

Not all of them. Just some. And always the same pages, although they don't seem to have anything special. The problem occurs with `pages/style.md`, `pages/navigation.md`, `pages/pages`, `pages/math_and_code.md`, and `src/jupyter/notebookipynb.ipynb`, and `pages/appendix.md`. The contents of each page is not sent to the corresponding `index.html` in `__site/pages/`.

This happens both with `serve()` and with `optimize`. But only locally. It runs fine in gh-actions and the generated site in https://github.com/rmsrosa/booksjl-franklin-template/ is fine, with all the pages rendered correctly.

I tried it with an M1 Mac and also with an old Intel Mac with an older MacOS. I tried with Julia 1.8, 1.7.1 and 1.7.2. All show exactly the same problem. I also tried going back to an old commit and with an older version 0.10.70 of Franklin, which I knew that used to work before, but they don't work anymore. I am puzzled.

Here is the rendering of one of the pages:
<img width="796" alt="Screen Shot 2022-03-14 at 11 48 23" src="https://user-images.githubusercontent.com/17986148/158203811-513da64f-cbd0-422e-b888-e6ecee2dd5e3.png">

**Comment by tlienart** [comment]
Can you tell me which version of `LiveServer` and make sure you're on `0.7.3`, if so, can you try using 0.7.1 ?

also can you do `serve(clear=true)`

**Comment by tlienart** [comment]
btw, you still have this jupyter stuff (`nbconvert`) going on which makes it hard for me to build your repo (it crashes) so I can't check myself...

maybe you can provide a toggle to remove all that stuff and facilitate debugging.

```
┌ Franklin Warning: in <menu_bar.md>
│ Encountered an issue processing 'menu_bar.md' in nklin-template/pages.
│ Verify, then re-start the Franklin server.
│ The error is displayed below:
│ ProcessFailedException(Base.Process[Process(`jupyter nbconvert --ExecutePreprocessor.timeout=-1 --to notebook --execute /Users/tlienart/Desktop/tjd/booksjl-franklin-template/generated/notebooks/weaved/weavejmd.ipynb --allow-errors --output /Users/tlienart/Desktop/tjd/booksjl-franklin-template/generated/notebooks/weaved/weavejmd.ipynb`, ProcessExited(1))])
└
ERROR: failed process: Process(`jupyter nbconvert --ExecutePreprocessor.timeout=-1 --to notebook --execute /Users/tlienart/Desktop/tjd/booksjl-franklin-template/generated/notebooks/weaved/weavejmd.ipynb --allow-errors --output /Users/tlienart/Desktop/tjd/booksjl-franklin-template/generated/notebooks/weaved/weavejmd.ipynb`, ProcessExited(1)) [1]
```

**Comment by rmsrosa** [comment]
Ok. I will make a branch without this. I guess it is from Weave.

**Comment by rmsrosa** [comment]
> Can you tell me which version of LiveServer and make sure you're on 0.7.3, if so, can you try using 0.7.1 ?

Apparently it is 0.7.3 in Manifest.toml:
```toml
[[deps.LiveServer]]
deps = ["Crayons", "FileWatching", "HTTP", "Pkg", "Sockets", "Test"]
git-tree-sha1 = "3c04368686c010fd6cb7b9a2d871b68457de244c"
uuid = "16fef848-5104-11e9-1b77-fb7a48bbb589"
version = "0.7.3"
```
This is a Franklin dependency. Is there an easy way to change its version?

Also, using `serve(clear=true)` has the same problem.

**Comment by rmsrosa** [comment]
Ok, I published the branch `nojupyter`. I didn't change `utils.jl` and didn't remove the `Weave` and `Literate` packages, but I removed the pages that needed them and the flags that triggered jupyter, so `jupyter` shouldn't be called. (Actually it might just suffice to set all the variables for page and binder in `config.md` to false, in the current version). Let me know if that works.

**Comment by tlienart** [comment]
thanks I'll check this out.

For LiveServer, just pin it to 0.7.1 in your global environment

**Comment by rmsrosa** [comment]
> For LiveServer, just pin it to 0.7.1 in your global environment

Yeah, that did it. It works with that! I went back and forth and it worked fine with 0.7.1 but not with 0.7.3. Thanks!

But what now? I hope you find out the reason for that.

**Comment by tlienart** [comment]
Issue is with LiveServer not with Franklin.

The new stuff from 0.7.1 is to have it able to show the content of a directory too and not just websites. I guess in changing this I broke something subtle which you're now hitting and needs fixing.

I'm glad you found it soon after the release and that you have something I should be able to reproduce so hopefully will get that sorted quickly

**Comment by tlienart** [comment]
ok so the issue is with the trailing backslash, so if you try with `0.7.3` and navigate to

* http://localhost:8000/pages/navigation    > it won't work (show directory content)
* http://localhost:8000/pages/navigation/  > it will work

Fix coming

**Comment by tlienart** [comment]
this should be fixed with `LS 0.7.4` thanks for the heads up, please let me know if you experience something like it again (basically if you see a dir listing or like a plain text file, it's a bug on LS side, it shouldn't happen now but just in case you experience that again, hopefully not!)
